ci/homebrew/ci-deploy.sh

```diff
@@ -11,14 +14,22 @@ else
 fi
 
 version=$(python3 setup.py --version)
-url="https://pypi.org/packages/source/g/git-machete/git-machete-$version.tar.gz"
+url="https://$pypi_host/packages/source/g/git-machete/git-machete-$version.tar.gz"
 sha256=$(
-  curl -s https://$pypi_host/pypi/git-machete/$version/json \
+  curl -s https://$pypi_host/pypi/git-machete/"$version"/json \
   | jq --raw-output '.urls | map(select(.packagetype == "sdist")) | .[0].digests.sha256')
 
 if [[ $do_push == true ]]; then
-  brew brew bump-formula-pr --url $url --sha256 $sha256 git-machete
+  brew bump-formula-pr --url "$url" --sha256 "$sha256" git-machete
 else
   echo "Refraining from push since it's a dry run"
-  brew bump-formula-pr --dry-run --url $url --sha256 $sha256 git-machete
+  brew bump-formula-pr --write-only --url "$url" --sha256 "$sha256" git-machete
+fi
+
+brew install --build-from-source --formula /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git-machete.rb
```

Comment:
Wow so homebrew-core is itself considered a _tap_... I always thought taps are only the custom repos like `VirtusLab/homebrew-git-machete` of ours 😯